src/main/kotlin/org/jetbrains/kotlinx/jupyter/repl.kt

```diff
@@ -360,6 +362,22 @@ class ReplForJupyterImpl(
         else context.compilationConfiguration.asSuccess()
     }
 
+    /**
+     * Used for debug purposes.
+     * @see ReplCommand
+     */
+    private fun printVars(isHtmlFormat: Boolean = false) = log.debug(
```

Comment:
Let's don't use `vars` word, only `variables`?

Reply:
Vars part of me hardly accepted the proposition, but accepted nevertheless